sf_stats.Api/sf_stats.Domain/Dtos/SeasonDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace sf_stats.Domain.Dtos
+{
+    public class SeasonDto
+    {
+        public int Id { get; set; }
+        public string Code { get; set; }
+        public string DisplayName { get; set; }
+    }
```

Comment:
Wondering if season should have a date range for start and end date

Reply:
added